.github/workflows/progress-tracker.yml

```diff
@@ -72,17 +73,18 @@ jobs:
             core.info(`Tracking progress of ${totalTrackedTasks} tasks across chapters`)
 
             const calcStats = nums => {
-              if (!nums.length) {
+              const size = nums.length
+              if (!size) {
                 return [0, 0, 0, 0, 0]
               }
               nums.sort((a, b) => a - b)
-              const half = Math.floor(nums.length / 2)
+              const sum = nums.reduce((a,b) => a + b, 0)
               return [
-                nums.reduce((a,b) => a + b, 0),
-                Math.min(...nums),
-                Math.max(...nums),
-                (nums.reduce((a,b) => a + b, 0) / nums.length).toFixed(2),
-                nums[half]
+                sum,
```

Comment:
Just realized this doesn't work for unique contributors.

For example, in https://github.com/bazzadp/almanac.httparchive.org/issues/73 there are only 2 people (you and Barry) but the All/Total cell says 4. I'd expect this value to also deduplicate repeated contributors.

Reply:
You don't want to deduplicate there as that is a chapter-wise stats and it represents a different quantity. The summary table is not showing in the Total columns what you feel it should. It has always been different. For example, even before we introduced `All`, `Reviewers` were adding up to 3, though there were only two unique people involved.

Reply:
In short, we are deduplication multiple roles of a person in a single chapter, but we are not deduplicating roles of individuals across chapters, and we should not. If you are looking for an absolute number of unique contributors and in how many chapters they each contributed in one or more roles, the place you should be looking at is under the Contributor Details under the the All section.